starter-code/clue.js

```diff
@@ -1,103 +1,155 @@
+    occupation: 'Entrepreneur',
+  },
+  {
+    firstname: 'Doctor',
+    lastname: 'Orchid',
+    color: 'white',
+    description: 'PhD in plant toxicology. Adopted daughter of Mr. Boddy',
+    age: 26,
+    image: 'http://www.radiotimes.com/uploads/images/Original/111967.jpg',
+    occupation: 'Scientist',
+  },
+  {
+    firstname: 'Victor',
+    lastname: 'Plum',
+    color: 'purple',
+    description: 'Billionare video game designer',
+    age: 22,
+    image: 'https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg',
+    occupation: 'Designer',
+  },
+  {
+    firstname: 'Kasandra',
+    lastname: 'Scarlet',
+    color: 'red',
+    description: 'She is an A-list movie star with a dark past',
+    age: 31,
+    image: 'https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg',
+    occupation: 'Actor',
+  },
+  {
+    firstname: 'Eleanor',
+    lastname: 'Peacock',
+    color: 'blue',
+    description:
+      'She is from a wealthy family and uses her status and money to earn popularity',
+    age: 36,
+    image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+    occupation: 'Socialité',
+  },
+  {
+    firstname: 'Jack',
+    lastname: 'Mustard',
+    color: 'yellow',
+    description:
+      'He is a former football player who tries to get by on his former glory',
+    age: 62,
+    image: 'https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg',
+    occupation: 'Retired Football player',
+  },
+];
+var weapons = [
+  {
+    type: 'Rope',
+    weight: 10,
+  },
+  {
+    type: 'Knife',
+    weight: 8,
+  },
+  {
+    type: 'Candlestick',
+    weight: 2,
+  },
+  {
+    type: 'Dumbbell',
+    weight: 30,
+  },
+  {
+    type: 'Poison',
+    weight: 2,
+  },
+  {
+    type: 'Axe',
+    weight: 15,
+  },
+  {
+    type: 'Bat',
+    weight: 13,
+  },
+  {
+    type: 'Trophy',
+    weight: 25,
+  },
+  {
+    type: 'Pistol',
+    weight: 20,
+  },
+];
+var rooms = [
+  {name: 'Dinning Room'},
+  {name:'Conservatory'},
+  {name:'Kitchen'},
+  {name:'Study'},
+  {name:'Library'},
+  {name:'Billiard Room'},
+  {name:'Lounge'},
+  {name:'Ballroom'},
+  {name:'Hall'},
+  {name:'Spa'},
+  {name:'Living Room'},
+  {name:'Observatory'},
+  {name:'Theater'},
+  {name:'Guest House'},
+  {name:'Patio'},
+];
+
+//console.log(rooms)
+//console.log(weapons)
+//console.log(characters)
 
+function random_selector(array) {
+  var index = Math.floor(Math.random() * array.length);
+  return array[index];
+}
+//console.log(random_selector(weapons))
 
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
 
-*/
 
-function random_selector(array) {
 
+function pick_mistery() {
+  var mystery = {};
+  mystery.killer = random_selector(characters);
+  mystery.location = random_selector(rooms);
+  mystery.weapon = random_selector(weapons);
+  return mystery;
```

Comment:
You can also do something like this if you want:

```
mystery: {
  killer: random_selector(characters),
  location: random_selector(rooms),
  weapon: random_selector(weapons)
}
```

By the way, I liked that you named the properties `killer` and `location` because they have much more meaning than `character` and `room` 😊 .